setup.py

```diff
@@ -27,13 +27,10 @@
 import textwrap
 
 
-if sys.version_info[:2] < (2, 7) or (3, 0) <= sys.version_info[:2] < (3, 4):
-    raise RuntimeError("Python version 2.7 or >= 3.4 required.")
+if sys.version_info[:2] < (3, 5):
+    raise RuntimeError("Python version >= 3.5 required.")
```

Comment:
We probably also want to add `python_requires`:
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
indeed, would be good to add `python_requires='>=3.5'`.

Reply:
Done.